Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/GetMetadataCommand.cs

```diff
@@ -41,7 +41,7 @@ namespace Yubico.YubiKey.Piv.Commands
     ///   GetMetadataResponse metadataResponse = connection.SendCommand(metadataCommand);<br/>
     ///   if (metadataResponse.Status == ResponseStatus.Success)
     ///   {
-    ///       PivKeyMetadata keyData = metadataResponse.GetData();
+    ///       PivMetadata keyData = metadataResponse.GetData();
```

Comment:
"PivKeyMetadata" didn't work when I tested this code, but "PivMetadata" did.

Reply:
Correct